src/libs/ck-libs/ampi/ampi.C

```diff
@@ -3692,9 +3698,11 @@ void ampi::static_mprobe(ampi *dis, int t, int s, MPI_Comm comm, MPI_Status *sts
 #endif
 }
 
+// Returns whether there is a message that can be received (return 1) or not (return 0) 
 int ampi::iprobe(int t, int s, MPI_Comm comm, MPI_Status *sts) noexcept
```

Comment:
Should this be a static function (as well as improbe)? @evan-charmworks

Reply:
I don't think that is necessary, because they don't have anything like `dis = dis->blockOnRecv();` where the `this` pointer would be invalidated.